packages/design-system/src/stitches.config.ts

```diff
@@ -225,11 +226,13 @@ export const {
       2: "10px",
       3: "15px",
       4: "20px",
-      5: "25px",
+      5: "28px",
       6: "35px",
       7: "45px",
       8: "65px",
       9: "80px",
+      sidebarLeft: "45px",
+      sidebarRight: "240px",
```

Comment:
all of these could be a nummeric value, I copied this from radix's design system example